vscode-dotnet-sdk-extension/src/extension.ts

```diff
@@ -139,7 +139,13 @@ export function activate(context: vscode.ExtensionContext, extensionContext?: IE
         ...eventStreamObservers);
 }
 
-function setPathEnvVar(pathAddition: string, displayWorker: IWindowDisplayWorker) {
+function setPathEnvVar(pathAddition: string, displayWorker: IWindowDisplayWorker, environmentVariables: vscode.EnvironmentVariableCollection) {
+    // Set PATH for VSCode terminal instances
+    if (!process.env.PATH!.includes(pathAddition)) {
+        environmentVariables.append('PATH', path.delimiter + pathAddition);
+    }
+
```

Comment:
I wonder, if you set process.env.path here is it isolated to this extension or do others benefit? if it's the latter we should include that too

Reply:
Based on the documentation I don't think setting process.env here would have any effect: https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_env

Reply:
Hmm, I guess it depends on if all extensions in VSCode share the same process (which they might). If that's the case wouldn't O# see the path update and be able to use it?

Reply:
Oh I assumed they wouldn't be on the same process but I can add that line in case. It will help ensure we don't add the PATH more than once anyways.